reflex/components/radix/themes/color_mode.py

```diff
@@ -55,44 +57,87 @@ def create(
         )
 
 
-class ColorModeSwitch(Switch):
-    """Switch for toggling light / dark mode via toggle_color_mode."""
+LiteralPosition = Literal["top-left", "top-right", "bottom-left", "bottom-right"]
 
-    @classmethod
-    def create(cls, *children, **props):
-        """Create a switch component bound to color_mode.
+position_values = get_args(LiteralPosition)
 
-        Args:
-            *children: The children of the component.
-            **props: The props to pass to the component.
+position_map = {
+    "position": position_values,
+    "left": ["top-left", "bottom-left"],
+    "right": ["top-right", "bottom-right"],
+    "top": ["top-left", "top-right"],
+    "bottom": ["bottom-left", "bottom-right"],
+}
 
-        Returns:
-            The switch component.
-        """
-        return Switch.create(
-            *children,
-            checked=color_mode != LIGHT_COLOR_MODE,
-            on_change=toggle_color_mode,
-            **props,
-        )
 
+# needed to inverse contains for find
+def _find(const, var):
+    return Var.create_safe(const).contains(var)
 
-class ColorModeButton(Button):
-    """Button for toggling chakra light / dark mode via toggle_color_mode."""
+
+def _set_var_default(props, position, prop, default1, default2=""):
+    props.setdefault(
+        prop, cond(_find(position_map[prop], position), default1, default2)
+    )
+
+
+def _set_static_default(props, position, prop, default):
+    if prop in position:
+        props.setdefault(prop, default)
```

Comment:
maybe i should just stop nitpicking this PR, but it just looks like these two could be combined now into a single `_set_default`

```python
def _set_default(props, position, prop, default1, default2=""):
    if isinstance(prop, Var):
        props.setdefault(
            prop, cond(_find(position_map[prop], position), default1, default2)
        )
    else:
        if prop in position:
            props.setdefault(prop, default1)
```

actually w/e lets just ship it 😂

Reply:
we can always refactor them in a follow up yeah 👍